charts/gardener/provider-local/registration/templates/controller-registration.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
j0pe7+msPLQSfbIgNR0n7iQSC+PEDkFw/Hn1SRfdi0e0RMaUstATuaWWXifadEBo79gl4+zKHnCTq7gF8tj5utOmrsP2+o5syfwvZQ3VqQsjTpgLxrOi6GKzzLO6Od/vFhWqfME1uoWDcPmqpcR9inIx1dFbjPfVgzQzx6BkuYSJVx06nfrrMFlhTQONRzOYEqBFpeKiRW+/V8jgBc9EOV2WewXFVIooTsEZAb35q4HMatmRKOCnX6iXSkWuHmGrtfWs5mTl21GxJbCZ0r7t3JBSeYt2Qna3ea9WeNW6NrjPnamc3VtRKte14+YuYe/eVL5+bWtsvdN4AB5lsEwvr22AVbmUNUZphSLue3Y7fPce8Hp1lBN5L537U4WDb+fGKtHUtU0dqLs3RWvVYRaqb3fkch+9Jf4FiWLNDeqw0WLn43VI0pAOLfJdTWzz6lDWhL4wxsHTBSG3Iv5bS83A7zdc51hbWxJLlzmVNlGMxx5J6uj2agt8NPS1Fh0/JyqpKjeifVaiaXNTKrJPslxYW3+DNR21X7TX4RJtVyF6J7LWYCkYXFxNVZHNc7QpLPOa5Tg0guh7U2muuuZ6c0jWS81v8VpTsd10tRT5ahc31a1oGft36kLM7YNXZ6PTs6ubs1dnJ/SpkZuL0euzyXh0cqacK7DQ6x/BZsqf88xc7DlXeJZPFen8yEYavXrKwvuaupLe89ejn87eArGXVzeXb8+u3l2dX5doNZF45iG7qdSpvLq0abjWHlvXH0z/XnEIWcsrFHY5rVaOJ9cfptYelx6E1OwEvHTWzWdVqhiuNtRzx/yR8476puiIKvBcmN8ky7FMD2zLL55kNf5AvrD4jW7hSpvEtCOHrzsB5SO07q6fBPWctMT7deehFfxP36iCqVyZytQ5I9hGjUoZKH1E6TQJ6Ts5wBlOQnfZ58xSEslnH7CdsNPl3Dkn20OVD2NFr+h2/4w/dJTfKcvq7Ji07kZ6oRZK71uAPvFLmfxZm2JTtLEt7rWrFWISEI/MV79QGtv542VqWLD9WCubgK2ZpXTevA2PVJ5J73W/VMKaA+2v9XZnPezi/wVxAhMlTNhbkNPEmePtHMG15z+D4v1P+NCc/x4EhBDPY/SUOqGqnJrPkFG8Ah4wt0nnzpiCtSQdxmPinKbs8QNjj4f2HO/n0K3wlUbJtLbDn+3I/RK0yk7yL3yCNvUJ7nAEVBv/cTwonv92j7uN/B8CVKmOUitIpwG1sOSnh0Fj1R28vVSLrY0hZ7rbmntkSm1ttgzLoL3cE8OyKcTYjD09twb7Y4/d1wC7yH84tex9HgKvu/9BL3sU3v/qHhmN/B8CNE3Lrexsjq0kXoAM/s5fcizqAhF4cUW8quCvtev7Lit3mHh0r6DRS2s/hSQJRPzG2kcjWoWwZQ2J22lRS4aBywOQipQO8ECc8Iz0cKH4VS2Ud9hXpqnFKx3Wm7LUyty1nvucZcMWZyo6TM0ttjt1I/7hntor7FOQfkr4BZTSwDpbjKhyuBOVEh6epHXn/WXC+NMGTpqaJ6L9vF1GbhOYUtdXGbyMlxmgBWx2iCup3Q2hTGfGLc8rxDwVDhGrx/W+dhDbbfaHRnawD9O0xlpB5xUc8bpy7k1XtUDgKsKoZPg4plyaGRHlYUgt/ajwtTODEfTc3yWDwU7ej/nHCMPIx6ns8hN/UUqcoAumTMNWZEV2SpD/osoXPeJRPqlZorjFD7UjVa+A7sClBOpqp6LMBzgtUcr6jUz5B9hRZx86/Joq8HMSM5Ug/IW593BEm9AkWcphZc8KulmumAK2SXO3EgfxXogeWQGTscppozXrUdHbYPRWxiYGABwxbLY86LUszqe8iPyzFqcf+JD/aWsUNCGOEuQQbKAQSpVXzxp6YJP6G4g/WwhzUZciyuJh9tSPbYE08Jiwi/2/a9y3hLr9/7FR9P/1jnrN+48Hga2Du7/+uM9NYeXbxW0qkealY8na2IXNAehbHiPuPP97yL8wh7ZXA3Xy3+8W338YGMfN/v8gsEn+pZnxqE78bV4XeOxB/IJhF/m/C6y9fgewRv6Nfun9Z+N40Nz/OggUDvHpFIugwMKRX7u8sYNdV1tojLdiRzcmzijd0W3/eiw0u6XykIt1Bd3SUFBjuvNp3KBQAnT4BpknK49F8PD1755/J8IDlq4/ou8OYCUsnD1nAGSmv8bVXk+LniHQoR79jS1Zrb2hC8VqSqyGfDZhDwJ41X1oEDWL7/1LQyWNq6u6U0TTS/eK6sI5oAB3bKnTx1N4BIdiFVHS1cJ6Vu5LOYt9DKjX/zy26HN+ALZG//eNXun3XwfN+c9hgN+mYZpS/sSHiXAiX1NK2QOYgvqish8hy7y/ndJdGPZDZsxmoDIerHttqaUGXdAXaMovaGV31obdb2kF6UdEHz+1QOQp5WK9SsMDq3R+WYMz/T1MX6ORCvVowN6h2ax22nSNa7daSkA4LaSeKaVLjRrZz8NKlWOTNaUMXoofPG3AlD9y2lCQnxttKFB5DLWhPHWVbyBe7BQ3IECI31OAwTR6x/RHFHWjLVeP4mWDdGqVOzmbhmXdbRe5X2iVQ/ZN9K9/twoB+Cyt4KRIUTxBVQGP9K3SJ0j+6oTJPstYx8BKIn6JgBkZLA+GiuG9UqRv7saLZKrbZNnJAjDVj1OPTDtLi7opOtPE9ZwOQ905JTbMMnthh+NWZVoKNCFzD99kt+94Xc1aOkcDUY2Jb7uvd9siIf1ZZkM3DP3Dl90ro9Sr9n99T3vW4xm6rrdaOR+LmXE0d8YMBn2RJKO8DfpmZauVxZ6zd92MVit/x89U1BO95mfKI8HWkyeAL2YSyEJexPXQFwjrcx1F8r7pdAXFsKNl10GhZksUptgBD9eH6QVTWZdlpupS3mAVotWSz+WV9Gn6W2BmSx2yNhsAqU5TfYb4EvDYq1oDDTTQQAMNNNBAAw000EADDTTQQAMNNNBAAw000EADDTTQQAMN/L3g/wHJycKUAKAAAA==
```

Comment:
Why do we have a diff here? Can you revert it, please?

Reply:
Sure, will revert. Just fyi, I suspect this has changed through local setup scripts